python/datasources/acs_health_insurance.py

```diff
@@ -0,0 +1,612 @@
+            male_state_data = gcs_to_bq_util.load_values_as_json(
+                gcs_bucket, self.get_filename(Sex.MALE, None, False))
+            female_state_data = gcs_to_bq_util.load_values_as_json(
+                gcs_bucket, self.get_filename(Sex.FEMALE, None, False))
+            male_county_data = gcs_to_bq_util.load_values_as_json(
+                gcs_bucket, self.get_filename(Sex.MALE, None, True))
+            female_county_data = gcs_to_bq_util.load_values_as_json(
+                gcs_bucket, self.get_filename(Sex.FEMALE, None, True))
+        else:  # LOAD DATA FROM ACS (useful for local debug)
+            male_state_request_params = format_params(
+                HEALTH_INSURANCE_BY_SEX_GROUPS_PREFIX, HEALTH_INSURANCE_BY_SEX_MALE_SUFFIXES)
+            female_state_request_params = format_params(
+                HEALTH_INSURANCE_BY_SEX_GROUPS_PREFIX, HEALTH_INSURANCE_BY_SEX_FEMALE_SUFFIXES)
+            male_county_request_params = format_params(
+                HEALTH_INSURANCE_BY_SEX_GROUPS_PREFIX, HEALTH_INSURANCE_BY_SEX_MALE_SUFFIXES, True)
+            female_county_request_params = format_params(
+                HEALTH_INSURANCE_BY_SEX_GROUPS_PREFIX, HEALTH_INSURANCE_BY_SEX_FEMALE_SUFFIXES, True)
+
+            male_state_data = self.get_acs_data_from_variables(
+                male_state_request_params)
+            female_state_data = self.get_acs_data_from_variables(
+                female_state_request_params)
+            male_county_data = self.get_acs_data_from_variables(
+                male_county_request_params)
+            female_county_data = self.get_acs_data_from_variables(
+                female_county_request_params)
+
+        # Aggregate and accumulate data in memory
+        self.accumulate_acs_data(male_state_data)
+        self.accumulate_acs_data(female_state_data)
+        self.accumulate_acs_data(male_county_data)
+        self.accumulate_acs_data(female_county_data)
+
+    '''
+    Takes data in the form of
+    [
+        [C27001A_002E, C27001A_003E, C27001B_002E, C27001B_002E][state][?county],
+        [12345, 1245, 123546, 124567, 01, 02]
+        ...
+    ]
+    This method determines the variables at the top (aka: key_row)
+    matches the value with the metadata from the prefix_suffix list
+    and stores it in the self.data as a tuple
+
+    (state_fip, county_fip, age, sex, race) example:
+    {
+        "('01', None, '0-6', 'Male', None)": {
+            "Total": "177643",
+            "With": "172850",
+            "Without": "4793"
+        },
+        "('01', None, '6-18', 'Male', None)": {
+            "Total": "414407",
+            "With": "400460",
+            "Without": "13947"
+        },
+        ...
+    } Note: This can be debugged via the total_health_insurance.json output file via local_debug
+    '''
+
+    def accumulate_acs_data(self, data):
+        key_row = data[0]
+        for row in data[1::]:
+            data = {}
```

Comment:
It would be clearer to use a different name for the local variable "data", as it's already the name of an argument passed into this method